scripts/install_backend_python_libs.py

```diff
@@ -24,10 +24,18 @@
 import subprocess
 import sys
 
-import pkg_resources
 import python_utils
 from scripts import common
 
+import pkg_resources
+
+IGNORED_LIBRARY_NAME_MISMATCHES = (
```

Comment:
Can you add a TODO to remove this and explain: (a) what the trigger for removal is, (b) what exact steps should be done for the removal?

Reply:
Done, although I just added identical TODOs everywhere that need to be cleaned up instead.